agora/Agora/AuthorityToken.hs

```diff
@@ -105,14 +105,20 @@ singleAuthorityTokenBurned gatCs txInfo mint = P.do
   let gatAmountMinted :: Term _ PInteger
       gatAmountMinted = psymbolValueOf # gatCs # mint
 
+  txInfoF <- pletFields @'["inputs"] $ txInfo
+
   foldr1
     (#&&)
     [ ptraceIfFalse "GAT not burned." $ gatAmountMinted #== -1
     , ptraceIfFalse "All inputs only have valid GATs" $
```

Comment:
It might be a good idea to rename this trace message. It's quite confusing what is incorrect about the transaction.